src/ButtonPanel.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => {
+  const renderButton = i => <Button name={i}>{ i }</Button>;
+
+  return (
+    <div>
```

Comment:
- [ ] **[OPTIONAL]** We can refactor this code by dynamically creating buttons if you have created an object containing arrays of button values:

```js
const buttons = { "group-1": ['A/C', '+/-', '%', '/'], "group-2": .....}
```
Then you can use [`object.keys`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys) for the outer loop and `map` for the inner loop to dynamically create the buttons.